tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -106,70 +104,72 @@ user creates release for publication
     user waits until page contains element  xpath://a[text()="Edit release summary"]  60
     user waits until h2 is visible  Release summary  60
     
+user opens publication on the admin dashboard
+    [Arguments]  
+    ...  ${publication}    
+    ...  ${theme}=%{TEST_THEME_NAME}    
+    ...  ${topic}=%{TEST_TOPIC_NAME}
+    
+    user navigates to admin dashboard
+    user selects theme and topic from admin dashboard  ${theme}  ${topic}
+    user waits until page contains accordion section   ${publication}   %{WAIT_120_SECONDS}
+    ${accordion}=  user opens accordion section  ${publication}
+    [Return]  ${accordion}
+        
 user creates methodology for publication
-    [Arguments]  ${publication}
-    user selects theme and topic from admin dashboard  %{TEST_THEME_NAME}  %{TEST_TOPIC_NAME}
-    user waits until page contains accordion section   ${publication}
-    user opens accordion section  ${publication}
-    user clicks button when available   Create methodology
+    [Arguments]
+    ...  ${publication}
+    ...  ${theme}=%{TEST_THEME_NAME}
+    ...  ${topic}=%{TEST_TOPIC_NAME}
+
+    ${accordion}=  user opens publication on the admin dashboard   ${publication}  ${theme}  ${topic}
+    user clicks button   Create methodology  ${accordion}
     user waits until h2 is visible  Methodology summary
     user checks summary list contains   Title   ${publication}
     user checks summary list contains   Status  Draft
     user checks summary list contains   Published on  Not yet published
     
+user views methodology for publication
+    [Arguments]  ${publication}     ${methodology_title}=${publication}
+    ${accordion}=  user opens publication on the admin dashboard   ${publication}
+    user opens details dropdown  ${publication}  ${accordion}
```

Comment:
I think this line should use `${methodology_title}` instead of `${publication}`

Reply:
Right you are - good spot!